File: Software Technologies/Exam/Skeletons/JavaScript Solution/models/Product.js
const mongoose = require('mongoose');

let productSchema = mongoose.Schema({
    name: {type: 'string', required: 'true'},
    status: {type: 'string', required: 'true'},
    quantity: {type: 'Number', required: 'true'},
    priority: {type: 'Number', required: 'true'},
});

let Product = mongoose.model('Product', productSchema);

module.exports = Product;

File: Software Technologies/Exam/Skeletons/C# Solution/ShoppingList/Models/Product.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace ShoppingList.Models
{
    public class Product
    {
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Status { get; set; }

        [Required]
        public int Quantity { get; set; }

        [Required]
        public int Priority { get; set; }
    }
}